### ISSUE TITLE
Initramfsless boot?

### ISSUE THREAD

**Issue by fixapc** [openzfs/zfs#13525]
Is it possible to import a pool with an initramfs so I can boot zfs root using the built in module ?

**Comment by mschilli87** [comment]
Yes, at least when `/boot` is not zfs. But I doubt this is the right place to ask for individual support. Unless you think you found a bug, you'd probably be better off asking for help on the [mailing list](https://openzfs.github.io/openzfs-docs/Project%20and%20Community/Mailing%20Lists.html).

PS: Don't forget to link your cross-post and close this issue though. :wink:

**Comment by fixapc** [comment]
ahh sorry about that misclick, if one of you guys can post me a link that would be nice. I will close this and correct no zfs boot just zfs root

**Comment by Dragon-debian** [comment]
Discussions tab is on the right side of Pull requests tab.
https://github.com/openzfs/zfs/discussions

I think its possible for those with write access to this repository to move an issue to discussions.
